nautobot/docs/user-guide/core-data-model/dcim/virtualdevicecontext.md

```diff
@@ -0,0 +1,3 @@
+# Virtual Device Context
+
+# TODO
```

Comment:
We'll also want to make sure this feature is documented in the 2.4 release note overview as well.

Reply:
https://github.com/nautobot/nautobot/issues/6249